packages/hebao_v1/contracts/stores/SecurityStore.sol

```diff
@@ -55,6 +39,7 @@ contract SecurityStore is GuardianStore
         uint    minInternval
         )
         external
+        onlyWalletModule(wallet)
```

Comment:
This one can actually be removed, `requireWalletModule(wallet);` is called only when `lastActive` is updated.

The reason I did this is to optimize for the most common case  `lastActive` doesn't need to updated, in which case there's no need to check `onlyWalletModule` (which is pretty expensive because it needs a CALL + SLOAD) because it acts just like a `view` function. And first calling a `view` function and then an actual `touch()` function is of course also more expensive because of the multiple calls.

So definitely a non-standard way to do things to save gas...

Reply:
If we remove `requireWalletModule(wallet)`, any address will be touch the wallet to prevent inheritance from happening, won't it?

Reply:
I noticed the requireWalletModule inside the funciton body.